server/game/cards/06-CotE/KakitaDojo.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../drawcard.js');
+const { DuelTypes } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class KakitaDojo extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Initiate a military duel',
+            initiateDuel: context => ({
+                type: DuelTypes.Military,
+                resolutionHandler: (winner, loser) => this.resolutionHandler(context, winner, loser)
+            })
+        });
+    }
+
+    resolutionHandler(context, winner, loser) {
+        if(loser) {
+            if(winner.hasTrait('duelist')) {
+                this.game.addMessage('{0} loses the duel and is bowed and cannot trigger its abilities until the end of the conflict', loser);
+                this.game.actions.jointAction(
```

Comment:
I don't think this should be a joint action - Borderlands Defender should be unable to use her Duelist training after losing this duel